modules/pool/pool.gql

```diff
@@ -1,12 +1,13 @@
 extend type Query {
-    poolGetPool(id: String!): GqlPoolBase!
+    poolGetPool(id: String!, chain: GqlChain): GqlPoolBase!
     poolGetPools(
         first: Int
         skip: Int
         orderBy: GqlPoolOrderBy
         orderDirection: GqlPoolOrderDirection
         where: GqlPoolFilter
         textSearch: String
+        userAddress: String
     ): [GqlPoolMinimal!]!
```

Comment:
seems like it's a search param, but there is already `where`. is it ok to move it there?

Reply:
fair